install_config/install/prerequisites.adoc

```diff
@@ -797,6 +797,45 @@ See
 link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Logical_Volume_Manager_Administration/index.html[Logical
 Volume Manager Administration] for more detailed information on LVM management.
 
+[[managing-docker-container-logs]]
+*Managing Docker Container Logs*
+
+If your container's log (*_/var/lib/docker/containers/HASH/HASH-json.log_*) is
+increasing, you can use `json-file` logging driver options to manage this.
+
+For example, `--log-opt max-size` rolls over the size, and you can specify the
+`--log-opt max-file`, which is the maximum number of files that a log is rolled
+over before being discarded. You can use these options with Docker 1.8.2 or
+later.
+
+Setting in *_/etc/sysconfig/docker (max-size=1M, max-file=3)_* and restart docker
+service:
+====
+----
+OPTIONS='--insecure-registry=172.30.0.0/16 --selinux-enabled --log-opt max-size=1M --log-opt max-file=3' <1>
+----
+====
+<1> `json.log` rollover by 1M and it stored 3 files.
```

Comment:
@ahardin-rh

From [docker's doc](https://docs.docker.com/engine/admin/logging/overview/)

~~~
Logs that reach max-size are rolled over. You can set the size in kilobytes(k), megabytes(m), or gigabytes(g).
~~~

You can say `1MB`.